doc/building.md

```diff
@@ -4,37 +4,53 @@ This guide documents the Onyx build process. For example usage of this building
 
 Note that steps 0, 1 and 2 can be skipped if you have already done them before.
 
-## Step 0: Set up the build environment
+## Step 0: Get required packages
 
-First, you'll need to download the [minimal sysroot](https://storage.googleapis.com/onyx-stuffs/minimal-sysroot.tar.zst).
-This allows you to have the toolchain build against a sane libc.
-Be sure to export `SYSROOT=$PWD/sysroot` and `ONYX_ARCH=x86_64` (right now, only x86_64 is supported).
+You need the following packages and programs: mtools, xorriso, libfl2, ninja, gn, grub-mkrescue.
```

Comment:
I don't remember why libfl2 is needed, hrm...

Reply:
Will test building without libfl2.

Reply:
Apparently it is not needed indeed. In fact I didn't have libfl2-devel installed all this time.